kolena/_experimental/object_detection/dataset.py

```diff
@@ -367,6 +367,10 @@ def _iter_object_detection_results(
     dataset_df = dataset.download_dataset(dataset_name)
     dataset_df = dataset_df[["locator", ground_truths_field]]
     _validate_column_present(dataset_df, ground_truths_field)
+    while ground_truths_field in df.columns:
```

Comment:
Sneaking in a fix here -- this would previously fail if the `ground_truths_field` was already present in the inference data frame. This is safe to rename because we simply drop it after the metrics computation.